pkg/trace/config/config.go

```diff
@@ -131,6 +131,9 @@ type AgentConfig struct {
 
 	// OTLPReceiver holds the configuration for OpenTelemetry receiver.
 	OTLPReceiver *OTLP
+
+	// EnabledRemoteRates defines if priority rates should be fetched remotely
+	EnabledRemoteRates bool
```

Comment:
`RemoteRates` should do 🙂 
```suggestion
	// RemoteRates reports whether priority rates should be fetched remotely.
	RemoteRates bool
```
Can we give a bit more detail in the documentation? Where from are they fetched? Is "fleet intake" good terminology?

I also changed to "reports whether" on the wording. I've seen that as common (and have used it) for booleans. I hope it's ok with you.